testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -108,7 +111,10 @@
     "PRMNCRST",  # Amber ncrst with positions/forces/velocities
     "PRM_NCBOX", "TRJ_NCBOX",  # Amber parm7 + nc w/ pos/forces/vels/box
     "PRMNEGATIVE",  # Amber negative ATOMIC_NUMBER (Issue 2306)
-    "PRMErr1", "PRMErr2", "PRMErr3",  # Amber TOP files to check raised errors
+    "PRMErr1",
+    "PRMErr2",
+    "PRMErr3",
+    "PRMErr4",  # Amber TOP files to check raised errors
```

Comment:
Probably worth moving that comment up to the first PRMErr1?

Reply:
Done.